tools/preflight/preflight.sh

```diff
@@ -483,44 +523,44 @@ spec:
       readOnly: false
 EOF
 
-  kubectl wait --for=condition=ready --timeout=5m pod/"${RESTORE_POD}" &>/dev/null
+  kubectl wait --for=condition=ready --timeout=2m pod/"${RESTORE_POD}" &>> preflight-log-${TIMESTAMP}.out
```

Comment:
make it 3m.. need to choose max safe value here

Reply:
resolved